tests/test_items.py

```diff
@@ -43,18 +46,48 @@ def test_new_item_history():
     assert item_history.item_id == item_id
 
 
-@pytest.mark.skip(reason="Item.history() not implemented")
+
 def test_item_assign():
     """ Test item assign """
+
+    company = Company(
+        id=223,
+        name="Bad Company",
+        code="Bad Cmpny",
+        address="addr"
+    )
+
+    location = Location(
```

Comment:
@paulodamaso It doesn't seem like you need to create a ``location`` for this test, or? As I see it's ``Company -> Item -> Employee``, location is irrelevant.